### ISSUE TITLE
Feature Request: Add more shapes

### ISSUE THREAD

**Issue by Rajdave69** [excalidraw/excalidraw#6235]
Currently there is only a circle, rectangle and rotated rectangle, It would be nice if there were more shapes hidden inside a shapes menu.  Something like 

![image](https://user-images.githubusercontent.com/86198810/218515316-7ec60221-f801-441c-9994-976f11128baf.png)

(This is from MS paint)

**Comment by dwelle** [comment]
under consideration

**Comment by DarkSector** [comment]
Hello folks, thank you for the amazing tool! Any more thoughts on adding more shapes?

**Comment by Cool-Game-Dev** [comment]
I would love some more shapes, are there any updates on this?

**Comment by andrzejSulkowski** [comment]
Same

**Comment by teghnet** [comment]
Indeed, it would be nice to have more shapes, like a **hexagon** and maybe even a triangle and pentagon (the last is less useful and I get that it would open up discussions on which way to point the shapes by default)

oh, and the `[tab]` is quite useful so the shapes could show up here as well: 

<img width="521" height="523" alt="Image" src="https://github.com/user-attachments/assets/0730bdb6-95a0-47f1-9172-ef53d2c0a542" />

**Comment by Igorgro** [comment]
Tldraw has a very nice set of default figures

<img width="268" height="246" alt="Image" src="https://github.com/user-attachments/assets/4aef831a-fce6-40b3-88f5-296f632f7009" />
